Setup/Uninstall.php

```diff
@@ -40,14 +32,11 @@ class Uninstall implements UninstallInterface
      * Default constructor
      *
      * @param Config $config
-     * @param IntegrationServiceInterface $integrationService,
      */
     public function __construct(
-        Config $config,
-        IntegrationServiceInterface $integrationService
+        Config $config
```

Comment:
The whole function declaration can fit on one line now, so you could move this up if you want to.